examples/gm_one_degree_calibration/distributed_gm_one_degree_model_calibration.jl

```diff
@@ -1,175 +0,0 @@
-using Oceananigans
-using Oceananigans.Architectures: arch_array
-using Oceananigans.Units
-using Oceananigans.Grids: on_architecture
-using Oceananigans.Utils: WallTimeInterval
-using Oceananigans.BuoyancyModels: buoyancy
-using Oceananigans.Models.HydrostaticFreeSurfaceModels: VerticalVorticityField
-using ClimaOcean.NearGlobalSimulations: one_degree_near_global_simulation
-using ParameterEstimocean
-using ParameterEstimocean.Utils: map_gpus_to_ranks!
-using ParameterEstimocean.Observations: FieldTimeSeriesCollector
-using ParameterEstimocean.Parameters: closure_with_parameters
```

Comment:
we give up on the 1 degree calibration?

Reply:
I think so --- do you want to keep maintaining it? I don't think we have plans to return to it so it makes sense to declare now that we are not maintaining it (we can always access this code in git history if we want to). What do you think?

Reply:
agreed!